QuackNBOOM/QuackNBOOM/BombController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  BombController.swift
+//  QuackNBOOM
+//
+//  Created by Benoit Neriah R. on 3/7/18.
+//  Copyright © 2018 Neriah and Neenee. All rights reserved.
+//
+// BombController handles multiple bombs and how they respond to taps.
+
+import Foundation
+import SpriteKit
+
+class BombController {
+    // TODO: create an array of bombs
+    // TEMP: only one bomb for testing
+    let bomb = Bomb(imagePath: "Bomb")
+    
+    
+    
+    // Initializer sets all bombs offscreen
+    //
+    required init() {
+        bomb.pos = CGPoint(x: -10, y: -10)
+    }
+    
+    
+    // Shoots a bomb at the location
+    //
+    // - Parameter: location: the position of the tap on the screen
+    public func shootBomb(location: CGPoint){
+        bomb.pos = location
+        bomb.resetExplosionCountdown()
```

Comment:
Maybe change the name of this function to better represent what your using it for. Perhaps startExplosionCountdown() ?

Reply:
**This requested change has not been implemented.**
**Reason:** The current name of this function will make sense in future versions of the game.
More functionality is planned to be added to this function; handling which bomb in the array to shoot and a *SKAction* may be added to animate the bomb being shot (the SKAction depends on development time).